pyomo/core/expr/boolean_value.py

```diff
@@ -30,36 +30,43 @@ def _generate_logical_proposition(etype, _self, _other):
 
 
 def as_boolean(obj):
-    """
-    A function that creates a BooleanConstant object that
-    wraps Python Boolean values.
+    """A function that converts its argument to a Pyomo Boolean (logiacl) object.
```

Comment:
```suggestion
    """A function that converts its argument to a Pyomo Boolean (logical) object.
```
Typo reported to crate-ci